posts/jhipster-5-release.md

```diff
@@ -0,0 +1,218 @@
+
+New frontend framework, see [here](https://github.com/jhipster/generator-jhipster/issues/6044) for a global status.
+
+
+## Angular support
+
+The whole support for Angular has been improved in JHipster 5 with multiple optimizations and rework. The main changes are the focus on Angular 6 and the deletion of popup dialogs for the entity screens.
+
+Here is a detailed list of all the changes related to Angular:
+- JHipster is focusing on Angular 6, that means the support for the older AngularJS 1.x has been removed
+- As a side effect, the support for Gulp and Bower has been removed as well
+- Migration to Webpack 4 (both Angular and React), which gives a nice performance boost
+- The admin modules are now lazy loaded, that speeds up the initial load of the application
+- The folder structure of entities has been improved, especially for microservices as now they are grouped by services. You can use a new flag `--skip-ui-grouping` to retain the old behavior
+- The entity screen is not using popups anymore for creation/edition
+
+[![jhipster5-entity-screen](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/06/jhipster5-entity-screen.png)](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/06/jhipster5-entity-screen.png)
+
+
+## Misc:
+
+Here the miscellaneous changes:
+- Both Angular and React now uses Prettier to format code
+- Migration from PhantomJS to Jest as PhantomJS is not developed anymore. The frontend tests are now using the Headless Chromium browser provided by Puppeteer
+
+
+# Backend changes
+
+## Spring Boot 2
+
+On the backend side, the biggest change is the upgrade of Spring Boot to the latest major version. Spring Boot 2 was released on March 1st and upgrades all the major Spring dependencies, the main one being the newest Spring Framework 5.
+
+The JHipster team has put a lot of effort to upgrade the generated java classes to Spring Boot and the updated dependencies.
+
+
+Among the new cool features, we can list:
+
+* Java 8 is the baseline and Java 9 is supported
+* Spring web MVC is now an optional dependency
+* Gradle plugin has been rewritten
+* HTTP/2 for Tomcat, Undertow and Jetty
+* Security is auto-configured by Spring-Boot auto-configure (unless JHipster overrides it)
+* new Actuators endpoints
+
+
+### Micrometer 
+Micrometer is the new metrics library provided by Spring-cloud. It aims to be a facade for all the metrics backend and to replaces the dropwizwards-metrics library. 
+Because Dropwizard is already deeply integrated with JHipster, the team has decided to exclude micrometer from this version until they can provide a better support.
+You can follow the [issue on GitHub](https://github.com/jhipster/generator-jhipster/issues/7100).
+
+
+### Reactive programming with Spring Web Flux
+
+One of the biggest change introduced by Spring 5 and Spring Boot 2 is the reactive programming support with spring-webflux.
```

Comment:
Maybe a "what's missing" section at the end of the article for items like this?

Reply:
👍